src/xGadgetron/pGadgetron/tests/test1.py

```diff
@@ -9,37 +10,19 @@
 Options:
   -r, --record   record the measurements rather than check them
   -v, --verbose  report each test status
-'''
-
-## CCP PETMR Synergistic Image Reconstruction Framework (SIRF)
-## Copyright 2015 - 2017 Rutherford Appleton Laboratory STFC
-##
-## This is software developed for the Collaborative Computational
-## Project in Positron Emission Tomography and Magnetic Resonance imaging
-## (http://www.ccppetmr.ac.uk/).
-##
-## Licensed under the Apache License, Version 2.0 (the "License");
-##   you may not use this file except in compliance with the License.
-##   You may obtain a copy of the License at
-##       http://www.apache.org/licenses/LICENSE-2.0
-##   Unless required by applicable law or agreed to in writing, software
-##   distributed under the License is distributed on an "AS IS" BASIS,
-##   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-##   See the License for the specific language governing permissions and
-##   limitations under the License.
 
-"""
-Created on Tue Nov 21 10:17:28 2017
+{author}
 
-@author: Evgueni Ovtchinnikov
+{licence}
```

Comment:
this is clearly cleaner. However, it means the license info isn't actually inside the file. I don't think this is according to Apache 2 instructions (and it assumes the file will not be distributed on its own). 
Not very important for these tests of course, but I think best to keep as-was (feel free to argue!)

Reply:
1. lawyers need to talk to programmers more to understand that "requiring" paragraphs of licence text per-file is insane, irrespective of standaloneness
2. making "reasonable efforts" per-file (such as this new version) is legally sufficient, regardless of whether that contradicts what any licence text says
3. this file does actually need and contain the licence text via the `import`. Running `pydoc`/`help()` on this would show the full licence text
4. because of the above dependency, this file could not work standalone without deliberate modification to remove any reference to licence, which would be clearly illegal
5. "requiring" paragraphs of licence text per-file is insane
6. see above
7. decent programmers do not do avoidable duplication. decent lawyers do not do avoidable repetition. `codacy` doesn't like code duplication. Even as it stands, this PR fixes some, but not all the duplication
8. see 1, 5 and 6

Happy to hear any counter arguments!

Reply:
:-) seems you have strong opinions! I'm ok with this in these test files only. nowhere else though. sorry.

Reply:
groan